src/item.cpp

```diff
@@ -1218,24 +1244,28 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
         damage_instance ammo_dam = has_ammo ? curammo->ammo->damage : damage_instance();
         // @todo This doesn't cover multiple damage types
         int ammo_pierce     = has_ammo ? get_ranged_pierce( *curammo->ammo ) : 0;
-        int ammo_range      = has_ammo ? curammo->ammo->range      : 0;
+        //int ammo_range      = has_ammo ? curammo->ammo->range      : 0;
```

Comment:
```suggestion
```
Just delete instead of commenting out.

Reply:
Ah, I forgot about that.  I had intended to go back and see whether a new item should be added which listed the ammo range somehow, but that's probably a problem for another day.  I'm happy to delete it.